bin/generate_operators_precedence.php

```diff
@@ -0,0 +1,50 @@
+<?php
```

Comment:
The `bin` folder should be added in the list of folders ignored from exports in `.gitattributes` to avoid shipping this internal script in dist archives downloaded by composer.

Reply:
Good idea